src/cephadm/cephadm

```diff
@@ -2292,6 +2315,9 @@ def get_container_mounts(ctx, fsid, daemon_type, daemon_id,
             crash_dir = '/var/lib/ceph/%s/crash' % fsid
             if os.path.exists(crash_dir):
                 mounts[crash_dir] = '/var/lib/ceph/crash:z'
+            if daemon_type != 'crash' and if_log_to_journald(ctx):
```

Comment:
why is crash different?

Reply:
It is already different:
https://github.com/ceph/ceph/blob/23e6580c22c9aaba17589eae9cc1eb0b34d3d7df/src/cephadm/cephadm#L2097

I have not checked the code. But I suppose it does not use the logging infrastructure used by other daemons.